tools/common.ps1

```diff
@@ -200,3 +195,20 @@ function Initiate-Bugcheck {
     Write-Host "$NotMyFault -accepteula -bugcheck $Code"
     & $NotMyFault -accepteula -bugcheck $Code
 }
+
+function Generate-WinConfig {
+    param (
+        [Parameter()]
+        [string]$Arch,
+
+        [Parameter()]
+        [string]$Config
+    )
+
+    $global:WinArch = $Arch
+    $global:WinConfig = $Config
+    if ($Arch -eq "x64")     { $global:WinArch = "amd64" }
+    else                     { $global:WinArch = "arm64" }
+    if ($Config -eq "Debug") { $global:WinConfig = "chk" }
+    else                     { $global:WinConfig = "fre" }
```

Comment:
Maybe we should return a tuple/hash table/object rather than using globals? E.g. $Config.Arch seems better than $WinArch especially if a script is extended to build/test multiple configs.

Reply:
I'd rather leave it as is for now. They can be fixed up in the future if you really want/care.